prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    const fullname = 'Michael Suh';
+    console.log('Fullname:', fullname);
+    const numberOfPets = '0';
+    console.log('Number of Pets:', numberOfPets);
+    const likesVegetables = 'false';
+    Boolean(likesVegetables);
```

Comment:
For numbers and booleans you'll want to leave out the quotes so that the JavaScript engine doesn't interpret them as strings.

Reply:
![variables-edited](https://user-images.githubusercontent.com/97069037/150040648-4771bd5d-a881-4639-b1d4-c640153c30ba.png)